sly.el

```diff
@@ -1482,7 +1482,6 @@ Fall back to `sly-init-using-slynk-loader' if ASDF fails."
     "Hack and be merry!"
     "Your hacking starts... NOW!"
     "May the source be with you!"
-    "Take this REPL, brother, and may it serve you well."
```

Comment:
Would replacing brother with the gender neutral sibling work?

"Take this REPL, sibling, and may it serve you well."

I have seen "my sibling in Christ" as a way to make an expression gender-neutral. Maybe something like  'paren-loving sibling' would work better? 

Anyway, removing the message is still an improvement over the current situation